### PR TITLE
Send message only of packages that do not exist, not all.

### PR DESCRIPTION
**Pull request by pacospace**

### CODE REVIEW

**Review by bissenbay — COMMENTED** [review]
lgtm

File: producer.py [hunk 1]
```diff
@@ -210,6 +190,26 @@ def package_releases_update(
...
+                                package_name=package_name,
+                                package_version=package_version,
+                                index_url=package_index.url,
+                                component_name=COMPONENT_NAME,
+                                service_version=__service_version__,
+                            )
+                        )
+                    )
+                    _LOGGER.debug(
+                        "Package %r in version %r hosted on %r was scheduled to be published on topic %r",
```
Comment (fridex) [inline producer.py:206]: The message sounds misleading to me.

**Review by fridex — COMMENTED** [review]
One minor comment - I don't want what the message wants to capture in logs. Otherwise LGTM.

File: producer.py (hunk 1 above)
Comment (pacospace) [reply producer.py]: Updated in 8d6e76e943232fd50953de72011ec5a33fc561f9

**Review by fridex — APPROVED** [review]

**Pull request merged**